dlp/system-test/metadata.test.js

```diff
@@ -19,13 +19,15 @@ const {describe, it, before} = require('mocha');
 const cp = require('child_process');
 const uuid = require('uuid');
 const DLP = require('@google-cloud/dlp');
+const {Storage} = require('@google-cloud/storage');
 
 const dataProject = 'bigquery-public-data';
 const dataSetId = 'samples';
 const tableId = 'github_nested';
 const fieldId = 'url';
 
-const bucketName = process.env.BUCKET_NAME;
+const storage = new Storage();
+const bucketName = `test-bucket-${uuid.v4()}`;
```

Comment:
Can we include the name of this test in the name to make it easier to track down the culprit test if these buckets stop being automatically cleaned up?

Reply:
@jakubrauch Resolved the comments, thanks for pointing out. Re-requested for review